kimchi/src/verifier.rs

```diff
@@ -383,301 +383,313 @@ where
             combined_inner_product,
```

Comment:
make sure to check 
<img width="254" alt="Screen Shot 2022-03-10 at 10 22 12 AM" src="https://user-images.githubusercontent.com/1316043/157729510-34af1c26-cee7-4fa3-90b1-11ef52a81641.png">